osidb/models.py

```diff
@@ -286,19 +287,6 @@ class FlawHistory(NullStrFieldsMixin, ValidateMixin, ACLMixin):
     # this model is unused so we don't care that it's a CharField with null=True
     pgh_label = models.CharField(max_length=100, null=True)  # noqa: DJ01
 
-    class FlawHistoryState(models.TextChoices):
-        """allowable Bugzilla states"""
-
-        ASSIGNED = "ASSIGNED"
-        CLOSED = "CLOSED"
-        MODIFIED = "MODIFIED"
-        NEW = "NEW"
-        ON_DEV = "ON_DEV"
-        ON_QA = "ON_QA"
-        POST = "POST"
-        RELEASE_PENDING = "RELEASE_PENDING"
-        VERIFIED = "VERIFIED"
-
     class FlawHistoryResolution(models.TextChoices):
```

Comment:
@osoukup Should I remove this variable? It is unused because the `resolution` field in this class used `FlawResolution` instead of `FlawHistoryResolution`.

Reply:
Yes please, all the unused stuff out! :smile_cat: